torch_np/_ndarray.py

```diff
@@ -81,6 +81,18 @@ def base(self):
     def T(self):
         return self.transpose()
 
+    @property
+    def real(self):
+        return asarray(self._tensor.real)
+
+    @property
+    def imag(self):
+        try:
+            return asarray(self._tensor.imag)
+        except RuntimeError:
+            zeros = torch.zeros_like(self._tensor)
+            return ndarray._from_tensor_and_base(zeros, None)
```

Comment:
Why not simply `return asarray(zeros)`?

Reply:
At some point we'll need to rationalize these two forms, agree.
Basically, `asarray` is anything array-like in, array out; here we explicitly construct the tensor, so my fingers naturally typed this line. The line above, with `asarray(self._tensor.imag)`, should be changed to follow line 94.

Reply:
I mean, `asarray(Tensor)` has the same semantics as `_from_tensor_and_base(Tensor, None)`, so we can decide to always prefer the first one over the latter one for conciseness and consistency.

Reply:
Not always

```
In [2]: a = np.zeros(3)

In [3]: np.asarray(a) is a
Out[3]: True
```

Reply:
note that I suggested doing so when using `torch.Tensor`s.